controllers/admin/self-managed/UpdatePageUpdateOptionsController.php

```diff
@@ -49,23 +54,81 @@ protected function displayRouteInUrl(): ?string
         return Routes::UPDATE_PAGE_UPDATE_OPTIONS;
     }
 
+    public function saveOption(): JsonResponse
+    {
+        $upgradeConfiguration = $this->upgradeContainer->getUpgradeConfiguration();
+        $upgradeConfigurationStorage = $this->upgradeContainer->getUpgradeConfigurationStorage();
+
+        $name = $this->request->request->get('name');
+
+        $config = [
+            $name => $this->request->request->get('value'),
+        ];
+
+        $error = $this->upgradeContainer->getConfigurationValidator()->validate($config);
+
+        if (empty($error)) {
+            if ($name === 'PS_DISABLE_OVERRIDES') {
```

Comment:
We could wait for the PR https://github.com/PrestaShop/autoupgrade/pull/993 and thus use the new constants, same thing in getParams

Reply:
+1